Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:21.04 as builder
```

Comment:
Can we use alpine or other lightweight images? Also 21.04 is not LTS, so let's not use it, best to stick to LTS versions imo.